app/Http/Controllers/Controller.php

```diff
@@ -136,11 +141,13 @@ protected function getPartnerTransactions(Partner $partner, array $criteria)
         $categoryId = $criteria['category_id'];
 
         $transactionQuery = $partner->transactions();
-        $transactionQuery->where('description', 'like', '%'.$query.'%');
-        $transactionQuery->whereBetween('date', [$startDate, $endDate]);
```

Comment:
Why are you removing this line, @RoisNewVersion? This line is still needed.

Reply:
Oh man, Sorry, That was my mistake. I always running phpunit test when done change something, and it shows me green.
Thanks for your corrections.

Reply:
Sorry for late update, I was busy. May you review the code. Thanks.